src/Security/Http/Firewall/AbstractRefreshAccessTokenListener.php

```diff
@@ -38,7 +38,7 @@ public function setResourceOwnerMap(ResourceOwnerMap $resourceOwnerMap)
 
     public function supports(Request $request): ?bool
     {
-        return null;
+        return true;
```

Comment:
Doesn't this mean that now this listener will always be checked?

Reply:
I think yes. I would prepare commit, that disables the listener if there are no one resource owner, that set `refresh_on_expire` option to `true`